2356-api_key_management/administrator_opens_api_keys.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Administrator opens API Keys
+
+    @2337
+    Scenario: Administrator opens API Keys for an Institution
+        Given that the user is logged in as Administrator
```

Comment:
Kunne spesifisert `Institution Administrator`

Reply:
Eller dette er kanskje `Application Admin`?

Reply:
Vi har ikke en rolle som heter `Institution Administrator`. Vi har kun `Administrator` og `Application Administrator`. Det ble ytret ønske fra POene om å bruke `Administrator` istedenfor `Institution Administrator` for ikke så lenge siden.

Reply:
I grensesnitt til bruker ja, men i DynamoDB er det fortsatt `Institution-admin`. `Application Administrator` eksisterer ikke for andre enn de som er det. Men så lenge vi er enige om at `Administrator` = `Institution-admin` er det forsåvidt like greit

Reply:
Ja, mente i grensesnittet (og forsåvidt også i feature-beskrivelsene), men synes også det er litt lite eksplisitt ja. Men vi får minne oss på at Admin = Institution Admin da 😅